notebooks/en/fine_tuning_llm_grpo_trl.ipynb

```diff
@@ -0,0 +1,1304 @@
+{
```

Comment:
<p>we pass a list of reward functions to the trainer that we previously defined &gt;&gt;&gt; we pass the two reward functions we previously defined to the trainer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>